nd-calendar/nd-calendar.js

```diff
@@ -67,7 +67,7 @@ export default class NudeCalendar extends HTMLElement {
 					return [];
 				}
 
-				for (let d = new BetterDate(low), i = 0; d <= new BetterDate(high + dur.day); d.setDate(d.getDate() + 1)) {
+				for (let d = new BetterDate(low), i = 0; d <= new BetterDate(high * 1 + dur.day); d.setDate(d.getDate() + 1)) {
```

Comment:
Nit, just for readability: 
```suggestion
				for (let d = new BetterDate(low), i = 0; d <= new BetterDate(Number(high) + dur.day); d.setDate(d.getDate() + 1)) {
```

(do check if this works as well, but I'd be very surprised if not)

Reply:
Yes much better!